### DISCUSSION TITLE
help with if then statement in handlebars

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2340] [Help]
Hi, I am looking for some help writing an if-then that statement in handlebars.

some back story:
I am using Mysql to hold my data.
I am using the internal DB to use the option picker. to write to my SQL DB I use the add column section in the button and point the column to my SQL DB and the value to the options picker.  to further complicate things I have relational formatting from the first drop-down menu that will display the correct next drop-down menu then that next dropdown will display the correct fields. 
first, drop-down-menu I point to my SQL column UNITS 
the second drop-down menus I point to my SQL column ITEMS

however, I have two instances where when I select the first drop-down menu there is not a second dropdown menu to select from and it jumps straight into the form.  I do not want to add an extra selector for the user to have to input if I can help it. 

my thought is to have an if-then statement in handlebars that I can use in addition to my ITEMS script that pulls the item from the second dropdown.

example:
if (first drop down) = water treater then "Water Treater" 

any help would be great thank you. please let me know if you need more info.

here is a screenshot of my current all items scrip it has all of the drop-down menus back to back, so it will capture the active value. 
![image](https://user-images.githubusercontent.com/80058238/129073567-dbb6b926-349d-4f77-9753-de7b49d55bb5.png)

**Comment by joebudi** [comment]
I am tagging @aptkingston into this one as he is the best person to help you here.

**Comment by aptkingston** [comment]
Hey @AndrewJRamay. Do you need help making the if statement? I'm struggling to understand what you're trying to do, but I can help you write an if statement in handlebars.

A basic if statement just looks like:
`{{ #if Form.Fields.Country }}true{{ /if }}`

You can use an `else` clause as well, if you need to:
`{{ #if Form.Fields.Country }}true{{ else }}false{{ /if }}`

If I understand you correctly, you could definitely use that here to build up the list of all items depending on the value of another field.

**Reply by KernOil** [reply]
@aptkingston  I'm still a bit confused I'm sorry 
I get the first part {{ #if (the field you want to call) }} 
but where is the condition? for example, I want the box to equal 'Water Treater' would I put 'Water Treater' where you put true?
